deny.toml

```diff
@@ -0,0 +1,49 @@
+targets = []
+
+[advisories]
+db-path = "~/.cargo/advisory-db"
+db-urls = ["https://github.com/rustsec/advisory-db"]
+vulnerability = "deny"
+unmaintained = "warn"
+yanked = "warn"
+notice = "warn"
+ignore = []
+
+[licenses]
+unlicensed = "deny"
+allow = [
+    "Apache-2.0",
+    "Apache-2.0 WITH LLVM-exception",
+    "BSD-3-Clause",
+    "ISC",
+    "MIT",
+]
+deny = []
+copyleft = "deny"
+allow-osi-fsf-free = "neither"
+default = "deny"
+confidence-threshold = 0.8
+exceptions = []
+
+[bans]
+multiple-versions = "warn"
+# Lint level for when a crate version requirement is `*`
+wildcards = "deny"
+highlight = "all"
+allow = []
+deny = [
+    { name = "ring" },
```

Comment:
:wrench: should this have a comment saying why it's denied that way if the status changes one will know it's OK to include.  Could also provide an indicator of what other types of crates to avoid.

Reply:
Yeah just on a personal level I'm curious about why this is forbidden (although the license situation _does_ look... complicated).

Reply:
Thanks for the issue link, should we link to that issue in the toml file?  I think it will be hard to clearly capture in one or two sentences the disparity in opinions on that issue.

Reply:
Yep, done